kalite/distributed/static/js/distributed/contentrec/hbtemplates/content-nextsteps-lesson.handlebars

```diff
@@ -1,6 +1,6 @@
 
 <div class="row" class="content-nextsteps-lesson">
-	<a href="/learn/{{ path }}">
+	<a href="/learn/{{ path }}" id="content-nextsteps-lesson-link">
```

Comment:
This is a repeated element in the page, we shouldn't use an ```id``` attribute here - can you change it to a class?